source/Calamari.AzureAppService.Tests/Legacy/LegacyAppServiceIntegrationTest.cs

```diff
@@ -67,16 +67,7 @@ public async Task Setup()
 
             resourceGroupClient = resourcesClient.ResourceGroups;
 
-            var resourceGroup = new ResourceGroup(resourceGroupLocation)
-            {
-                Tags =
-                {
-                    // give them an expiry of 14 days so if the tests fail to clean them up
-                    // they will be automatically cleaned up by the Sandbox cleanup process
-                    // We keep them for 14 days just in case we need to do debugging/investigation
-                    ["LifetimeInDays"] = "14"
-                }
-            };
+            var resourceGroup = new ResourceGroup(resourceGroupLocation);
```

Comment:
I've removed these Lifetime tags as I've never seen anyone use the lingering services to debug and these just hang around for a couple of additional weeks.